### PR TITLE
makefile: Enable creating ExtensionService CRD yaml

### LINKED ISSUE #2711: Auth: Enable ExtensionService CRD
**Please describe the problem you have**

Enable installing the ExtensionService CRD.

xref #432 
xref #2643 
xref #2710

### PR DESCRIPTION
**Pull request by stevesloka**
Enables the creation of the ExtensionService CRD when running `make generate`.

Fixes #2711

Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: examples/contour/01-crds.yaml [hunk 1]
```diff
@@ -1,6 +1,241 @@
...
+            protocol:
+              description: Protocol may be used to specify (or override) the protocol used to reach this Service. Values may be tls, h2, h2c. If omitted, protocol-selection falls back on Service annotations.
+              enum:
+              - h2
+              - h2c
+              type: string
+            protocolVersion:
+              description: This field sets the version of the GRPC protocol that Envoy uses to send requests to the extension service. Since Contour always uses the v2 Envoy API, this is currently fixed at "v2". However, other protocol options will be available in future.
+              enum:
+              - v2
```
Comment (ccravens) [inline examples/contour/01-crds.yaml:53]: @stevesloka does this mean that ExtensionService does not yet support Envoy GRPC v3?

Comment (stevesloka) [reply examples/contour/01-crds.yaml:53]: yup that's correct, we're working on moving over to v3 right now.

**Review by skriss — APPROVED** [review]
LGTM 👍

**Comment by skriss** [comment]
Just noticed https://github.com/projectcontour/contour/blob/301fb4e7a26ad219d18d27fd2189a8f5ba70223f/cmd/contour/serve.go#L305-L309 while debugging with Chad, that TODO should be addressed

**Review by youngnick — APPROVED** [review]
LGTM

**Comment by stevesloka** [comment]
@skriss I removed the TODO and the informer is now always started up looking for the ExtensionService CRD.

**Review by skriss — APPROVED** [review]
:+1: LGTM!

**Pull request merged**